templates/base.html

```diff
@@ -17,10 +17,12 @@
             {% endblock %}
             {% block title_suffix %}
                 {% wagtail_site as current_site %}
-                 - {{ current_site.site_name }}
+                 – {{ current_site.site_name }}
```

Comment:
Is this now an en dash rather than a hyphen? It's hard to tell 😅

Reply:
Good eye! It's an en-dash yes, Ashley asked for it to be an en-dash rather than a hyphen - for content reasons